### PR TITLE
feat: add a script to perform inference using the Claude3 model

### PR DESCRIPTION
**Pull request by kentosasaki-jp**
### Description
Add an inference script for Claude3 (We use Claude3 as a model for the performance comparison of VLMs).

### Confirmed

- [x] I have updated the documentation accordingly.
- [ ] I have adhered to the coding standards and guidelines of this project.
- [ ] I have added comments, especially in hard-to-understand areas.

### CODE REVIEW

File: playground/scripts/inference_claude3.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import os
+import base64
+from pathlib import Path
+import json
+from anthropic import Anthropic
+from PIL import Image
+import io
```
Comment (Ino-Ichan) [inline playground/scripts/inference_claude3.py:7]: isortお願いします！

Comment (kentosasaki-jp) [reply playground/scripts/inference_claude3.py]: 対応しました。確認お願いします。

**Review by Ino-Ichan — APPROVED** [review]
LGTM!

**Review by ymgaq — COMMENTED** [review]
LGTM!

**Pull request merged**